src/server/server.cpp

```diff
@@ -2296,7 +2296,13 @@ void server::player_version_handler(
 			return;
 		}
 	}
-	*out << "Player " << parameters << " not found.";
+	if(parameters.empty()) {
```

Comment:
Wouldn't it make more sense to test for empty _before_ looping over all the players?

Reply:
it could but it wouldn't change behviour.